constants.py

```diff
@@ -27,15 +27,14 @@
 # MixUpVI training hyperparameters
 MAX_EPOCHS = 100
 BATCH_SIZE = 1024
-BATCH_KEY = ["donor_id", "assay"] # needs to be of type list
 TRAIN_SIZE = 0.7 # as opposed to validation
 CHECK_VAL_EVERY_N_EPOCH = None
 if TRAIN_SIZE < 1:
     CHECK_VAL_EVERY_N_EPOCH = 1
 # MixUpVI model hyperparameters
 CONT_COV = None  # list of continuous covariates to include
-ENCODE_COVARIATES = False  # should be always False for now, we don't encode cat covar
-ENCODE_CONT_COVARIATES = False  # True or False, whether to include cont covar
+CAT_COV = ["donor_id", "assay"] # list of categorical covariates to include
+ENCODE_COVARIATES = True # whether to encode cont/cat covars (they are always decoded)
```

Comment:
"they are always decoded" => Can you point to the part of the code where you've seen this please ?

Reply:
Yes! 
In the initialization of the VAE decoder [there](https://github.com/scverse/scvi-tools/blob/9b99f95be35120046c14ca69bd44f10c2d2188ee/scvi/module/_vae.py#L217) you can see that the categorical variables passed is cat_list. 
But in the initialization of the VAE encoder [there](https://github.com/scverse/scvi-tools/blob/9b99f95be35120046c14ca69bd44f10c2d2188ee/scvi/module/_vae.py#L179), what is given to the encoder is not exactly cat_list, but encoder_cat_list, which is equal to cat_list only when encoder_covariates is True.